tests/util_tests/test_qt_importer.py

```diff
@@ -107,7 +106,6 @@ def test_qt_importer_with_pyside_interface_qt4(self):
         assert qt.base["__name__"] is qt.binding_name
         assert qt.base["__version__"] is qt.binding_version
 
-    @skip_if_pyqt4(found=False)
```

Comment:
This tests seems to fail on CI. As the description says:

>         This test only runs if PyQt4 is available and PySide, PySide2 are not available.

Can we consider removing this unit tests entirely?